docs/introduction.rst

```diff
@@ -82,27 +82,66 @@ approaches to this:
    output for the related images, we can deduce certain aspects of the
    model's behavior.
 
+
+.. note:: These descriptions apply only to *using the model after it has been created*; nothing is implied about how the model is constructed.
+
+.. note:: This distinction applies to both the *concept* and *implementation*. A model could support white box reasoning but lack the APIs to access the internal state; conversely, it might provide complete API access yet provide no guidance on how that state influences the output. **API access to the model does not automatically imply white box XAI.**
+
 Let's take a look at the pros and cons of these two approaches.
 
 White Box Methods
 ^^^^^^^^^^^^^^^^^
 
-*Explanation options are correlated to how the model operation can be
-inspected and interpreted. pros: closely tied to the individual model, thus can leverage
-specific knowledge about its architecture. Measures the actual
-computation which generates the output. Cons: specific to individual
-models / classes of models; harder to generalize across models.
-Requires modifying the model implementation to gain access; may
-require updating as the implementation evolves.*
+A **white box** model exposes some (or all) of the internal state of
+the model; the explanation draws a connection between this exposed
+state and the model output. Some models are intrinsically introspective to the point where they are not so much "white box" as transparent:
+
+* In `linear regression
+  <https://en.wikipedia.org/wiki/Linear_regression>`_ the influence of
+  the observations on the solution are an immediate consequence of the
+  method.
+
+* In a `decision tree <https://en.wikipedia.org/wiki/Decision_tree>`_,
+  the output is directly computed by making the comparisons and
+  branches encoded in the model.
+
+An example of a white box explanation method for CNNs is Grad-CAM (`paper <https://arxiv.org/abs/1610.02391>`_, `code <https://github.com/ramprs/grad-cam/>`_), which exposes some (but not all) of the CNN layers to the explanation generation algorithm.
+
+.. figure:: figures/intro-grad-cam-annotated.png
+
+   Figure 5: Grad-CAM architecture. Functional regions annotated to align with Figure 4: input in green, CNN architecture in yellow, output in orange. The explanation functionality is in blue; the APIs granting access to the model are in red. Figure from the `Grad-CAM code repo. <https://github.com/ramprs/grad-cam/>`_
+
+In Grad-CAM, first the model computes its output per the typical processing flow: input (green), through the CNN (yellow), to the output (orange.) The explanation is created via an additional processing step which uses the output and feature maps from within the CNN (red circles) to measure and visualize the activation of those regions associated with the output (blue).
```

Comment:
processing step *that* uses the output

Reply:
"to measure and visualize the activation of those regions associated with the output (_blue_)" 
> I thought the output was orange?